libseccomp/src/lib.rs

```diff
@@ -830,10 +830,26 @@ mod tests {
             ScmpCompareOp::from_str("SCMP_CMP_NE").unwrap().to_native(),
             ScmpCompareOp::NotEqual.to_native()
         );
+        assert_eq!(
+            ScmpCompareOp::from_str("SCMP_CMP_LT").unwrap().to_native(),
+            ScmpCompareOp::Less.to_native()
+        );
         assert_eq!(
             ScmpCompareOp::from_str("SCMP_CMP_LE").unwrap().to_native(),
             ScmpCompareOp::LessOrEqual.to_native()
         );
+        assert_eq!(
+            ScmpCompareOp::from_str("SCMP_CMP_EQ").unwrap().to_native(),
+            ScmpCompareOp::Equal.to_native()
+        );
+        assert_eq!(
+            ScmpCompareOp::from_str("SCMP_CMP_GE").unwrap().to_native(),
+            ScmpCompareOp::GreaterEqual.to_native()
+        );
+        assert_eq!(
+            ScmpCompareOp::from_str("SCMP_CMP_GT").unwrap().to_native(),
+            ScmpCompareOp::Greater.to_native()
+        );
```

Comment:
Cloud you add `SCMP_CMP_MASKED_EQ` test?

Reply:
In adddtion, Could you add test in case of `ParseError` as follows.

```rust
 assert_eq!(
            ScmpCompareOp::from_str("INVALID_FLAG"),
            Err(SeccompError::new(ParseError))
        );
```

Reply:
@savithry-j Could you add `assert!(ScmpCompareOp::from_str("INVALID_FLAG").is_err());`?